Win32Emu/Win32/Modules/DDrawModule.cs

```diff
@@ -3266,154 +3254,6 @@ private uint DDraw_SetCooperativeLevel(ICpu cpu, VirtualMemory memory, uint ddra
 			return (uint)DDResult.DD_OK;
 		}
 
-		private uint DDraw_SetDisplayMode(ICpu cpu, VirtualMemory memory, uint ddrawHandle)
```

Comment:
Upon removing this function, did you check if there are any regressions in the Async function? Like, is there something in here the Async version doesn't have?